### PR TITLE
NEW view mode "Kanban" for list of societe

### PR DESCRIPTION
**Pull request by lamrani002**
# NEW|New view mode Kanban

### CODE REVIEW

File: htdocs/societe/list.php [hunk 1]
```diff
@@ -483,7 +482,7 @@
 	$sql .= ", sc.fk_soc, sc.fk_user";
 }
 // Add fields from extrafields
-if (!empty($extrafields->attributes[$object->table_element]['label'])) {
+if (empty($extrafields->attributes[$object->table_element]['label'])) {
```
Comment (Hystepik) [inline htdocs/societe/list.php:485]: to fix

File: htdocs/societe/list.php [hunk 2]
```diff
@@ -497,9 +496,10 @@
 
 $sql .= " FROM ".MAIN_DB_PREFIX."societe as s";
 $sql .= " LEFT JOIN ".MAIN_DB_PREFIX."societe as s2 ON s.parent = s2.rowid";
-if (!empty($extrafields->attributes[$object->table_element]['label']) && is_array($extrafields->attributes[$object->table_element]['label']) && count($extrafields->attributes[$object->table_element]['label'])) {
-	$sql .= " LEFT JOIN ".MAIN_DB_PREFIX.$object->table_element."_extrafields as ef on (s.rowid = ef.fk_object)";
+if (empty($extrafields->attributes[$object->table_element]['label']) && is_array($extrafields->attributes[$object->table_element]['label']) && count($extrafields->attributes[$object->table_element]['label'])) {
```
Comment (Hystepik) [inline htdocs/societe/list.php:499]: same as before

File: htdocs/societe/list.php [hunk 3]
```diff
@@ -508,7 +508,7 @@
 $sql .= ' LEFT JOIN '.MAIN_DB_PREFIX."c_stcomm as st ON s.fk_stcomm = st.id";
 // We'll need this table joined to the select in order to filter by sale
 if ($search_sale == -2) {
-	$sql .= " LEFT JOIN ".MAIN_DB_PREFIX."societe_commerciaux as sc ON sc.fk_soc = s.rowid";
+	$sql .= " LEFT JOIN ".MAIN_DB_PREFIX.$object->table_element."societe_commerciaux as sc ON sc.fk_soc = s.rowid";
```
Comment (Hystepik) [inline htdocs/societe/list.php:511]: old code was good

**Comment by lamrani002** [comment]
Now, it 's good, I fixed all problems

**Pull request merged**